### DISCUSSION TITLE
Welcome to automa Discussions!

### DISCUSSION THREAD

**Discussion by Kholid060** [AutomaApp/automa#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DavidHF** [comment]
trying to put key strokes like Tab and ctrl F into the action box. (Browser with Insta) I can get exactly no results. And I cant find any documentation what to enter to make it work... 

I wanted to detect/record a mouse click but that didnt work. So I tried Tab - no luck

**Comment by LeoUKR** [comment]
Please tell me if there are detailed instructions for this add.
